adal/__init__.py

```diff
@@ -27,7 +27,7 @@
 
 # pylint: disable=wrong-import-position
 
-__version__ = '0.2.0rc1'
+__version__ = '0.2.3rc1'

```

Comment:
We will finalize the version before release. For now the pattern is a little random, but it doesn't matter